ios/PurchasesHybridCommon/PurchasesHybridCommon/Purchases+HybridAdditions.swift

```diff
@@ -22,7 +22,7 @@ import RevenueCat
                           usesStoreKit2IfAvailable: Bool = false,
                           dangerousSettings: DangerousSettings?,
                           shouldShowInAppMessagesAutomatically: Bool = true,
-                          verificationMode: String? = nil) -> Purchases {
+                          verificationMode: String?) -> Purchases {
```

Comment:
The compiler didn't complain 😕  but I think this is needed to disambiguate both configure methods.